### ISSUE TITLE
Switch to true Free Open Source license

### ISSUE THREAD

**Issue by acostaja** [vlang/vinix#18]
Hi, respectfully i suggest to consider fully distribute the project under non-gpl licenses as it's demonstrated in project's like tensor flow (almost 3/4 AI projects are licensed in MIT-like licenses) it attract coders looking to get rid off viral clauses (as did tensorflow with R-based ML), while choosing for some license in Open Source community driven projects is seen with ideology or fear, other faces it with the practical issues derived from each license conditions, GPL is often considered as opesource savior as it prevents the IP being "stolen" by an commercial or rival actor, but actually it also kick off developers which just want some building blocks for his project's, mature developers understand how important it's for code life cycle to return to the community the improvements or corrections privately made, original developers will ever keep the advantage of the original know how and of being the community hub, while newcomers doesn't get discouraged to build a commercial product on top. Un mature developers will try to steal the code anyway with an fork (from gpl) or with an more elaborate "full" re-write with "own" IP.

Please don't see it ideologically, watch a what license has dominated most popular opensouce projects from last 10 years: non-gpl sound success: tensorflow, python, pytorch, node.js it's quite unusual seeing an new GitHub project based on python,node.js,swift even kotlin using gpl-like licenses.

I see big enthusiasm around V lang, but GPL IMHO kicked away more adopters than enthusiast it drives into.

j.a.

**Comment by vitalyster** [comment]
Trying to interpolate project "success" from the project license does not sounds correctly

**Comment by crthpl** [comment]
The most successful open source project, Linux, is GPL-Licensed

**Comment by acostaja** [comment]
Respectfully Linux success is more related to its architecture efficiency and flexibility over openbsd and the web hosting need for a "for-free" host operating system to run the Web, it succeed despite the GPL tax, for comparison Google (and industry in general) Nex all new OS: Fuchsia is opensouce licensed in MIT-like or Apache like licensed non-gpl, gpl vital terms are as toxic as what it wants to prevent.

**Comment by medvednikov** [comment]
@acostaja Vinix will also be reusing/translating Linux drivers, and those are GPL.

So anything else is not an option.
